### DISCUSSION TITLE
Leap motion controller 2 Support

### DISCUSSION THREAD

**Discussion by mata75** [Yellow-Dog-Man/Resonite-Issues#1137] [New Features]
It would be great if it supports hand tracking with Leap Controller camera Sensor

**Comment by LexiBasilisk** [comment]
Hi @mata75 - as ProbablePrime and Frooxius notified you in your issue #1138 - we already have Leap support- but it requires the SDK to be updated as indicated in #8.

I'll be closing this discussion. Thank you for the report.
